scripts/system/snapshot.js

```diff
@@ -15,6 +15,7 @@
 (function () { // BEGIN LOCAL_SCOPE
 Script.include("/~/system/libraries/accountUtils.js");
 var AppUi = Script.require('appUi');
+var ContextMenu = Script.require('contextMenu');
```

Comment:
A comment as to why we are requiring contextMenu even though we are not using it would be nice.
```suggestion
// The context menu conflicts with our controller mappings, so we disable it any time our app is open.
var ContextMenu = Script.require('contextMenu');
```